### ISSUE TITLE
audio player accessibility

### ISSUE THREAD

**Issue by tmakarios** [conversejs/converse.js#3382]
**Describe the bug**
Screen reader users can't play audio. 

**To Reproduce**
1. Use a screen reader and keyboard to go to the play button on an audio file in a chat, such as at https://freespoken.nz/bible-radio/
2. Press Enter
3. Discover that the play button is unavailable. 

**Expected behavior**
The audio file should play. 

**Environment (please complete the following information):**
 - Desktop or Mobile?  Windows 11 with NVDA, or JAWS
 - Browser [e.g. Chrome, Firefox] Not sure
 - Converse.js version [e.g. 4.2.0] 10.1.7, I think

**Additional context**
You can see the comments of the person who actually tested it in the replies to this toot https://theres.life/@TMakarios/112278231413032952

**Comment by unwaivering** [comment]
I'm actually the screen reader user who did the testing. I can confirm that this is the problem I'm having when using the web interface.

**Comment by unwaivering** [comment]
I've tried this in Firefox, and I was able to read the audio player controls. I've tried using the carrot browsing in NVDA, and it made the navigation much better. However while using the carrot browsing, it did not play. It did play without it. Perhaps you may want to test conversant JS to see if it complies with WCAG standards. you could do so by going here.
https://webaim.org/resources/evalquickref/

**Comment by jcbrand** [comment]
Converse doesn't do anything fancy with audio/video, it uses the built-in HTML `audio` and `video` elements.

The issue seems to be that browsers don't support accessibility features for these built-in elements. :roll_eyes: 

This article has more info:
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia 

Looks like the workaround is to create a custom interface.

**Comment by Karankhatik** [comment]
Hey, i want to work on this issue. Please assign it to me. Thanks.

**Comment by jcbrand** [comment]
@Karankhatik Thank you for offering to work on this. There is a $100 bounty for whoever provides a solution for it (see here: https://github.com/conversejs/converse.js/issues/2481).

However, we don't assign tickets to people, because often they ask for this and then disappear. 

Also, while only the person who resolves the issue gets the bounty, we don't restrict who is able to try and resolve it. If multiple people want to try in parallel, then they can.

If you want to work on this ticket, you can start by creating a pull request and mention this ticket in your PR.

**Comment by aliraza556** [comment]
@jcbrand Hi,

I've submitted PR #3924 which fixes the audio player accessibility issue. Screen reader users (NVDA/JAWS) were unable to interact with audio controls in chat messages. The fix replaces the native audio element with a custom accessible player that supports keyboard navigation and proper ARIA attributes.
Could you please review it when you have a chance?
Thanks!
